apps/web/src/utils/userUtils.ts

```diff
@@ -145,3 +145,17 @@ export const formatLocation = ({
 
   return intl.formatMessage(commonMessages.notProvided);
 };
+
+interface IsVerifiedGovEmployeeArgs {
+  isGovEmployee?: Maybe<boolean>;
+  workEmail?: Maybe<string>;
+  isWorkEmailVerified?: Maybe<boolean>;
+}
+
+export const isVerifiedGovEmployee = ({
```

Comment:
@esizer Does this function satisfy the acceptance criteria for https://github.com/GCTC-NTGC/gc-digital-talent/issues/12363 and then allow it to be closed as well?

Reply:
May need to wait and see. I think to but not 100% sure